latest_major_version.go

```diff
@@ -0,0 +1,143 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Not sure where to put this script - it should only be run in the github workflow, so maybe in a `.github/scripts` directory?

Reply:
maybe you can check out the `.github/workflows` directory. It contains the pipelines we run in CI, and the `app` subdirectory has another versioning go file that might be similar to what you're working with!